cob_twist_controller/include/cob_twist_controller/limiters/limiter_base.h

```diff
@@ -43,7 +44,29 @@ class LimiterBase
         /**
          * Pure virtual method to mark as interface method which has to be implemented in inherited classes.
          * The intention is to implement a method which enforces limits to the q_dot_out vector according to
-         * the calculated joint velocities and / or joint positions.
+         * the calculated joint velocities and / or joint positions and or cartesian twists.
+         * @param q_dot_ik The calculated joint velocities vector which has to be checked for limits.
+         * @param q The last known joint positions.
+         * @return Scaled joint velocities vector.
+         */
```

Comment:
as the LimiterBase class does not contain any pure virtual function anymore....remove this comment block
copy them to the LimiterJointBase and LimiterCartesianBase and adjust accordingly